.github/actions/unit-tests/action.yml

```diff
@@ -2,7 +2,7 @@ name: "Unit Tests"
 description: "Run unit tests using go"
 inputs:
   GO_VERSION:
-    default: "1.17"
+    default: "1.18"
```

Comment:
This is unused, but it seems like you are removing it in #356. Is that correct?

Reply:
Yes, this should be unused, but I used this PR to just update the only Go version everywhere and make changes to the GitHub action in the other ones to keep this one small.